forge/routes/api/project.js

```diff
@@ -84,45 +88,60 @@ module.exports = async function (app) {
         // If we have roles that limit creation, that will need to be checked here.
-        if (teamMembership) {
-            const team = teamMembership.get('Team')
-            const project = await app.db.models.Project.create({
-                name: request.body.name,
-                type: request.body.options.type || 'basic',
-                url: ''
-            })
-
-            // const authClient = await app.db.controllers.AuthClient.createClientForProject(project);
-            // const projectToken = await app.db.controllers.AccessToken.createTokenForProject(project, null, ["project:flows:view","project:flows:edit"])
-            // const containerOptions = {
-            //     name: request.body.name,
-            //     projectToken: projectToken.token,
-            //     ...request.body.options,
-            //     ...authClient
-            // }
-
-            await team.addProject(project)
-            await app.containers.create(project, {})
+        if (!teamMembership) {
+            reply.code(401).send({ error: 'Current user not in team ' + request.body.team })
+            return
+        }
 
-            await app.db.controllers.AuditLog.projectLog(
-                project.id,
-                request.session.User.id,
-                'project.created'
-            )
-            await app.db.controllers.AuditLog.teamLog(
-                team.id,
-                request.session.User.id,
-                'project.created',
-                { id: project.id, name: project.name }
-            )
+        const team = teamMembership.get('Team')
 
-            const result = await app.db.views.Project.project(project)
-            // result.meta = await app.containers.details(project);
-            result.team = team.id
-            reply.send(result)
-        } else {
-            reply.code(401).send({ error: 'Current user not in team ' + request.body.team })
+        const stack = await app.db.models.ProjectStack.byId(request.body.stack)
+
+        if (!stack) {
+            reply.code(400).send({ error: 'Invalid stack' })
+            return
         }
+
+        const project = await app.db.models.Project.create({
+            name: request.body.name,
+            type: '',
+            url: ''
+        })
+
+        // const authClient = await app.db.controllers.AuthClient.createClientForProject(project);
+        // const projectToken = await app.db.controllers.AccessToken.createTokenForProject(project, null, ["project:flows:view","project:flows:edit"])
+        // const containerOptions = {
+        //     name: request.body.name,
+        //     projectToken: projectToken.token,
+        //     ...request.body.options,
+        //     ...authClient
+        // }
+
+        await team.addProject(project)
+        await project.setProjectStack(stack)
+        await project.reload({
+            include: [
+                { model: app.db.model.ProjectStacks }
```

Comment:
```suggestion
                { model: app.db.models.ProjectStack }
```

Sorry, typo when I made the suggestion